### PR TITLE
Docs: run vela-core locally for debugging & breakpoints

### PR DESCRIPTION
**Pull request by fourierr**
Signed-off-by: fourierr <maxiangboo@qq.com>
Add docs about how to run vela-core locally for debugging & breakpoints.
ping @wonderflow

### CODE REVIEW

File: docs/contributor/code-contribute.md [hunk 1]
```diff
@@ -369,4 +369,14 @@ Learn how to write the docs by the following guide:
...
+
+## FAQ
+
+### Q: How can I run vela-core locally for debugging?
+ - Firstly, scaling the replicas of `kubevela-vela-core` to 0 for leader election of `controller-manager`, e.g `kubectl scale deploy -n vela-system kubevela-vela-core --replicas=0`.
+ - Secondly, removing the `WebhookConfiguration`, otherwise an error will be reported when applying your application using `vela-cli` or `kubectl`.
+   ```shell
+   Internal error occurred: failed calling webhook 'validating.core.oam.dev.v1beta1.applications': Post "https://vela-core-webhook.vela-system.svc:443/validating-core-oam-dev-v1beta1-applications?timeout=10s"
+   ```
+ - Finally, pulling the repository of KubeVela and debugging the code with breakpoints.
```
Comment (wonderflow) [inline docs/contributor/code-contribute.md]: What's the difference between the current make vela-core run in the section https://kubevela.io/docs/contributor/code-contribute#build

Comment (fourierr) [reply docs/contributor/code-contribute.md]: > This part should be in the code contribution gude

ok

File: docs/contributor/code-contribute.md [hunk 2]
```diff
@@ -151,6 +151,16 @@ Start to test.
 make e2e-test
 ```
 
+### Debug & breakpoints
+
+To run vela-core locally for debugging and breakpoints
```
Comment (wonderflow) [inline docs/contributor/code-contribute.md]:
```suggestion
To run vela-core locally for debugging with kubevela installed in the remote cluster:
```

File: docs/contributor/code-contribute.md [hunk 3]
```diff
@@ -151,6 +151,16 @@ Start to test.
 make e2e-test
 ```
 
+### Debug & breakpoints
```
Comment (wonderflow) [inline docs/contributor/code-contribute.md]:
```suggestion
### Debugging Locally with Remote KubeVela Environment
```

File: docs/contributor/code-contribute.md (hunk 1 above)
Comment (fourierr) [reply docs/contributor/code-contribute.md]: > What's the difference between the current make vela-core run in the section https://kubevela.io/docs/contributor/code-contribute#build

build is about how to run locally, it is mainly about how to debug locally

File: docs/contributor/code-contribute.md [hunk 4]
```diff
@@ -151,6 +151,16 @@ Start to test.
...
 
+### Debugging Locally with Remote KubeVela Environment
+
+To run vela-core locally for debugging with kubevela installed in the remote cluster:
+- Firstly, scaling the replicas of `kubevela-vela-core` to 0 for leader election of `controller-manager`, e.g `kubectl scale deploy -n vela-system kubevela-vela-core --replicas=0`.
+- Secondly, removing the `WebhookConfiguration`, otherwise an error will be reported when applying your application using `vela-cli` or `kubectl`.
+  ```shell
+  Internal error occurred: failed calling webhook 'validating.core.oam.dev.v1beta1.applications': Post "https://vela-core-webhook.vela-system.svc:443/validating-core-oam-dev-v1beta1-applications?timeout=10s"
+  ```
+- Finally, pulling the repository of KubeVela and debugging the code with breakpoints.
```
Comment (wonderflow) [inline docs/contributor/code-contribute.md]:
```suggestion
- Finally, you can use the commands in the above [Build](#build) and [Testing](#Testing) sections, such as `make run`, to code and debug in your local machine.
```

**Review by wonderflow — COMMENTED** [review]
写文档是很考验水平的，要上下文连贯，不能只考虑很小的一个点。

**Comment by fourierr** [comment]
> 写文档是很考验水平的，要上下文连贯，不能只考虑很小的一个点。

好的，最近打算翻一翻文档，也学习一下怎么写好文档

**Review by wonderflow — APPROVED** [review]

**Pull request merged**